dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/DeepStorageBrowserPresenter.java

```diff
@@ -335,7 +388,19 @@ private void logsORJobsMenuItemAction(final CheckMenuItem menuItem, final VBox v
         }
     }
 
-    public void logText(final String log, final LogType type) {
+    private void registerLoggingServiceListener() {
+        LOG.info("Registering loggingService observable");
+        final Observable<LoggingService.LogEvent> observable = loggingService.getLoggerObservable();
+        observable.doOnNext(logEvent -> {
+            if (Platform.isFxApplicationThread()) {
+                this.logTextForParagraph(logEvent.getMessage(), logEvent.getLogType());
+            } else {
+                Platform.runLater(() -> this.logTextForParagraph(logEvent.getMessage(), logEvent.getLogType()));
+            }
+        }).subscribe();
```

Comment:
Was this code block always like this?  Even if it was, we should change this.

JavaRX allows you to tell it which thread you want the handlers to run in when fired.  We should update this to so that it tells it to run in the fxapplicationthread, then we don't need to do this application thread check.